activerecord/test/schema/schema.rb

```diff
@@ -409,6 +409,14 @@
     t.references :student
   end
 
+  create_table :students, force: true do |t|
+    t.string :name
+    t.boolean :active
+    t.integer :college_id
+  end
+
+  add_foreign_key :lessons_students, :students, on_delete: :cascade
```

Comment:
Currently `schema.rb` for tests have three `add_foreign_key`.

* `add_foreign_key :authors, :author_addresses`  
https://github.com/rails/rails/blob/v5.0.0.rc1/activerecord/test/schema/schema.rb#L55-L66

```ruby
  create_table :authors, force: true do |t|
    t.string :name, null: false
    t.integer :author_address_id
    t.integer :author_address_extra_id
    t.string :organization_id
    t.string :owned_essay_id
  end

  create_table :author_addresses, force: true do |t|
  end

  add_foreign_key :authors, :author_addresses
```

* `add_foreign_key :fk_test_has_fk, :fk_test_has_pk, column: "fk_id", name: "fk_name", primary_key: "pk_id"`  
https://github.com/rails/rails/blob/v5.0.0.rc1/activerecord/test/schema/schema.rb#L997-L1004

```ruby
    create_table :fk_test_has_fk, force: true do |t|
      t.integer :fk_id, null: false
    end

    create_table :fk_test_has_pk, force: true, primary_key: "pk_id" do |t|
    end

    add_foreign_key :fk_test_has_fk, :fk_test_has_pk, column: "fk_id", name: "fk_name", primary_key: "pk_id"
```

* `add_foreign_key :lessons_students, :students`  
https://github.com/rails/rails/blob/v5.0.0.rc1/activerecord/test/schema/schema.rb#L407-L410  
https://github.com/rails/rails/blob/v5.0.0.rc1/activerecord/test/schema/schema.rb#L780-L784  
https://github.com/rails/rails/blob/v5.0.0.rc1/activerecord/test/schema/schema.rb#L1005

For a table that has a foreign key must be removed prior to the table that is referenced by a foreign key, I think it is important that the order of `create_table` that is dependent on foreign key is easy to understand.